packages/column-views/src/unit-details/panel.ts

```diff
@@ -215,12 +216,44 @@ function UnitDetailsContent({
       label: "Lithology",
       lithologies,
       features: lithologyFeatures,
+      onClick: (e) => {
+        const parent = e.target.parentElement;
+        const grandparent = parent.parentElement;
+        const greatgrandparent = grandparent.parentElement;
+        const check = [parent, grandparent, greatgrandparent].filter(
+          (el) => el.className.includes("lith_id")
+        );
+        const id = check[0]?.className.match(/lith_id-(\d+)/)?.[1];
+
+        window.open("/lex/lithology/" + id, "_blank");
+      }
     }),
     h(AgeField, { unit }, [
       h(Parenthetical, h(Duration, { value: unit.b_age - unit.t_age })),
-      h(IntervalProportions, { unit }),
+      h(IntervalProportions, { 
+        unit,
+        onClick: (e) => {
+          const child = e.target;
+          const parent = e.target.parentElement;;
+          const grandparent = parent.parentElement;
+          const greatgrandparent = grandparent.parentElement;
+
+          const check = [child, parent, grandparent, greatgrandparent].filter(
+            (el) => el.className.includes("int_id")
+          );
+
+          const id = check[0]?.className.match(/int_id-(\d+)/)?.[1];
+
+          window.open("/lex/intervals/" + id, "_blank");
+        },
+       }),
```

Comment:
This is overly complex and needs to be rethought. But the idea of being able to click on an `IntervalProportions` component and navigate to the proper interval is good.